mods/d2k/rules/arrakis.yaml

```diff
@@ -35,9 +35,7 @@ SANDWORM:
 			Dune: 100
 			Spice: 100
 	TargetableUnit:
-		TargetTypes: Underground
-	RevealsShroud:
-		Range: 32c0
+		TargetTypes: Ground
```

Comment:
I am not sure what you (and @Phrohdoh) mean by "above ground". Do you mean only when attacking? In the original you could attack worms if you see the trail that represents them. I like to think of it like they were as in Emperor: Battle for Dune when fremen are riding them - partially sticking out of the ground.